pyproject.toml

```diff
@@ -16,7 +16,7 @@ dependencies = [
     "numpy<2.0.0",
     "packaging",
     "pint",
-    "bluesky>=1.13.0a3",
+    "bluesky>=1.13.0a4",
```

Comment:
I have change this to make sure that we are at version the hardware test was conducted